plonky2/src/hash/arch/aarch64/poseidon_goldilocks_neon.rs

```diff
@@ -2,37 +2,24 @@
 
 use std::arch::aarch64::*;
 use std::arch::asm;
+use std::mem::transmute;
 
-use plonky2_field::field_types::Field64;
 use plonky2_field::goldilocks_field::GoldilocksField;
 use plonky2_util::branch_hint;
 use static_assertions::const_assert;
 use unroll::unroll_for_loops;
 
-use crate::hash::poseidon::{
-    Poseidon, ALL_ROUND_CONSTANTS, HALF_N_FULL_ROUNDS, N_PARTIAL_ROUNDS, N_ROUNDS,
-};
+use crate::hash::poseidon::Poseidon;
 
 // ========================================== CONSTANTS ===========================================
 
 const WIDTH: usize = 12;
 
-// The order below is arbitrary. Repeated coefficients have been removed so these constants fit in
-// two registers.
-// TODO: ensure this is aligned to 16 bytes (for vector loads), ideally on the same cacheline
-const MDS_CONSTS: [u32; 8] = [
-    0xffffffff,
-    1 << 1,
-    1 << 3,
-    1 << 5,
-    1 << 8,
-    1 << 10,
-    1 << 12,
-    1 << 16,
-];
-
-// The round constants to be applied by the second set of full rounds. These are just the usual round constants,
-// shifted by one round, with zeros shifted in.
+const EPSILON: u64 = 0xffffffff;
+
+// The round constants to be applied by the second set of full rounds. These are just the usual
+// round constants, shifted by one round, with zeros shifted in.
+/*
```

Comment:
This PR comments out a bunch of code in this file, which is now being compiled again. The commented code will be used in the partial rounds.